macros/get_equivalent_types.sql

```diff
@@ -0,0 +1,93 @@
+-- This macro returns a list of equivalent data types for a given standard type
```

Comment:
this macro is needed because the type checking dbt test just does a string match between the expected type and the type as listed in the database.